refactor(locale)!: use singular locale definition keys (#2932)
faker-js/faker: docs/guide/upgrading.md

### Renamed Locale Definitions

> The following locale definitions have been renamed to use singular keys and file names:

Reviewer: The information that the file names have changed should not be relevant to the end user. I think a more generic description would be fitting here.

```suggestion
The following locale definitions have been adjusted to align with fakers locale definition naming standard:
```

The title of this section should propably also be changed too 🤔

Author: I have run but I am getting this error " ERR_PNPM_READ_FROM_STORE  Unexpected token '', ""... is not valid JSON".
I am figuring out the error after that I will update.

> The following locale definitions have been adjusted to align with fakers locale definition naming standard:

Reviewer 2: ```suggestion
The following locale definitions have been adjusted to align with Faker's locale definition naming standard:
```

> | Old                                         | Replacement                                  |

Reviewer 3: The other tables use locercase headers as well.

```suggestion
| old                                         | replacement                                  |
```
